Library/Formula/swiftgen.rb

```diff
@@ -4,11 +4,12 @@ class Swiftgen < Formula
   url "https://github.com/AliSoftware/SwiftGen/archive/0.5.0.tar.gz"
   sha256 "555f190f2ffef940eebd80a926eeb05d3d0de573412028c5bd2184e2b9542929"
   head "https://github.com/AliSoftware/SwiftGen.git"
+  revision 1
 
-  bottle do
-    cellar :any
-    sha256 "29391f183e1606e50008ad148b3665d240c28ab5cbd42a293d1ca99b29aa3793" => :el_capitan
-    sha256 "c150775bf2f6b8e77a821eae7bd89233a134eb22ce5eed88de9eaf2822dd79ee" => :yosemite
+  def pour_bottle?
```

Comment:
> Is there any guarantee that the machine BrewBot uses to build the bottle use an Xcode named /Applications.Xcode.app all the time?

On stable OS X versions, yes. On very beta OS X versions it may well be `Xcode-beta.app` but 10.9, 10.10 & 10.11 are all stable. The first runs Xcode 6.4, the latter two run 7.0.1, all three are `/Applications/Xcode.app`.